src/sections/about.html

```diff
@@ -0,0 +1,17 @@
+<div>
+  <p class="font-body font-normal pt-36 pb-16 px-10 text-black text-2xl md:pr-8 md:pl-36 md:text-3xl md:px-16 lg:leading-[3.5rem] lg:text-[2.875rem] ">
```

Comment:
Remove the empty spaces after Tailwind CSS classes, it is not a good practice to use arbitrary values